treeherder/perf/tasks.py

```diff
@@ -10,3 +16,49 @@ def generate_alerts(signature_id):
     newrelic.agent.add_custom_parameter("signature_id", str(signature_id))
     signature = PerformanceSignature.objects.get(id=signature_id)
     generate_new_alerts_in_series(signature)
+
+
+@task(name='check-confirming-perf-alerts')
+def check_confirming_perf_alerts_status():
+    confirming_interval = settings.PERFHERDER_CONFIRMING_INTERVAL
+
+    def traceback_producer_job_type(perf_alert):
+        push = perf_alert.summary.push
+        signature = perf_alert.series_signature
+
+        associated_perf_datum = PerformanceDatum.objects.filter(push=push, signature=signature).first()
+        return associated_perf_datum.job.job_type if associated_perf_datum else None
+
+    def extract_pushes_to_check(many_nearby_pushes, alert_push, push_range=2):
+        many_nearby_pushes = list(many_nearby_pushes)
+
+        alert_push_idx = None
+        for index, push in enumerate(many_nearby_pushes):
+            if alert_push.id == push.id:
+                alert_push_idx = index
+                break
+
+        from_push = alert_push_idx - push_range
+        from_push = from_push if from_push > 0 else 0
+        to_push = alert_push_idx + push_range
+
+        return list(many_nearby_pushes[from_push:to_push])
+
+    confirming_alerts = PerformanceAlert.objects.filter(status=PerformanceAlert.CONFIRMING)
+    for alert in confirming_alerts:
```

Comment:
This looks like it could be expensive in terms of queries. Could we test on prototype first?

Reply:
I don't think it should be very expensive, as should never have all that many confirming alerts afaik.

Reply:
(though I agree testing on prototype is a good idea regardless)